### DISCUSSION TITLE
Language Files

### DISCUSSION THREAD

**Discussion by oksuzgil** [mynaparrot/plugNmeet-server#375] [Ideas]
Language files change

Hello, I am having a problem with the language files. Just as changes were made to the css files in the "Design Customization" section, can we import the language files we specified, if any? For example, in the waiting room "Please wait until the moderator approves you." but I want to change it to "Please wait until your educator approves you." While I changed it to "your instructor" someone else might want to change it to "your teacher". Instead of making file changes in every plugnmeet update, I think we can import a copy of the plugnmeet language file by editing it, use the language file we imported first, use the plugnmeet language file if it is not in our language file, so that if there is a new change, we can use it without any errors until I add it myself.

Or there must be a way to add language packs from outside, in accordance with the structure of the system (except for compilation on our own side).

I would be glad if you could evaluate it.

**Comment by jibon57** [comment]
The solution is simple, maintain your own repo & upstream whatever you think is important. Also you can create your own script to copy only important JS & CSS files from release zip file too & avoid to entirely replace `assets` directory directly. Language files will always update in our GitHub repo.

**Comment by oksuzgil** [comment]
You say it very well, but after completing the server installation, I update the server with bash. In this case, the language files will also change. When we choose to use the language file we marked externally while developing the project (currently PHP for me), none of these situations will occur. Additionally, by setting up and expanding a single plugnmeet server, we can easily change the language in multiple different projects. We can organize and use it according to which text structure will be used in which project. (as if making a design change)
